docs/application/dotnet/getting-started/runtime.md

```diff
@@ -0,0 +1,24 @@
+<a id="runtime"></a>
+# Managed Runtime Advantages
+
+A managed runtime offers the following advantages to your application development:
+
+- **Faster development**
+
+  Application development is accelerated since the managed runtime handles many functions that otherwise have to be coded into the application. The Tizen Native API and Tizen Web API already provide this benefit.
```

Comment:
Application development is accelerated as the managed runtime handles many functions that otherwise have to be coded into the application. The Tizen Native API and the Tizen Web API already provides this benefit.

Reply:
done